index.html

```diff
@@ -470,6 +470,33 @@ <h4>jsonSchema</h4>
                  }
                  </pre>
                </p>
+               <p>
+                The <code>$ref</code> property can be used to reference another jsonSchema without redefining the JSON structure.
+                <pre class="example jsonschemacredential-jsonschema nohighlight" title="Example JsonSchema Credential with jsonSchema">
```

Comment:
You can't have a `<pre>` inside a `<p>`. There should be a `</p>` following what is now line 474, and the one now in line 499 should be removed. I see there's another above this, and there may be more in the document. Do you want a PR for this, an issue for this, or is this note sufficient?

Reply:
This note is sufficient i'll update it

Reply:
I have updated all instances I found of this, and a number of other HTML lint issues.

Do you know what HTML formatter is used by default for these specs? I keep having to manually format it.

I tested with Prettier and it ended up causing a full file diff, so i reverted to manually formatting.

Reply:
I know nothing of the formatter. Perhaps it's somewhere in GitHub Actions?